addon/components/rdf-input-fields/concept-scheme-selector.js

```diff
@@ -88,4 +96,43 @@ export default class RdfInputFieldsConceptSchemeSelectorComponent extends InputF
     this.hasBeenFocused = true;
     super.updateValidations();
   }
+
+  getFieldOptionsAsObject(fieldOptions) {
+    if (typeof fieldOptions == 'string') {
+      try {
+        return JSON.parse(fieldOptions.trim());
+      } catch (error) {
+        console.error(` | Could not parse string options to json object`);
+        console.error(` | `, error);
+
+        return {};
+      }
+    }
+
+    return fieldOptions;
+  }
+
+  isValidFieldOptions(fieldOptions) {
+    let config = this.getFieldOptionsAsObject(fieldOptions);
```

Comment:
This was already  called in the `loadOptions` method, so this one shouldn't be needed.

Reply:
Removed that method